README.md

```diff
@@ -79,7 +79,46 @@ To install the project, clone it :
 git clone git@github.com:planet-learning/planet-learning.git
 ```
 
-In order to make it run on [TESS](https://fr.wikipedia.org/wiki/Transiting_Exoplanet_Survey_Satellite) data, you need to download it from [here](http://archive.stsci.edu/tess/bulk_downloads/bulk_downloads_ffi-tp-lc-dv.html).
+#### Getting the required data
+
+In order to make it run on [TESS](https://fr.wikipedia.org/wiki/Transiting_Exoplanet_Survey_Satellite) data, you need to download : 
+
+- TESS TIC catalog files from https://archive.stsci.edu/tess/tic_ctl.html
+- TESS light curves from http://archive.stsci.edu/tess/bulk_downloads/bulk_downloads_ffi-tp-lc-dv.html
+
+#### NFS
+
+As the amount of TESS data needed to run this project is important, it is not stored on the computing machine but on a dedicated machine with a big storage hosting a NFS server.
+
+The folder structure on the NFS must be the following :
+
+```py
+.
+└── data/
+    ├── catalog/ # Extract the catalog files here
+    ├── confirmed/
```

Comment:
@proxyma-centauri can you explain how to retrieve the list of confirmed exoplanets?

Reply:
Done in 7ba8791